test/integ/routers/alpha-router/alpha-router.integration.test.ts

```diff
@@ -268,6 +274,19 @@ describe('alpha router integration', () => {
     const aliceAddress = await alice.getAddress();
     expect(aliceAddress).toBe(alice._address);
 
+    const MixedRouteQuoterV1Factory =
+      await hardhat.hre.ethers.getContractFactoryFromArtifact(
+        MixedRouteQuoterV1_ABI,
+        alice
+      );
+    const MixedRouteQuoterV1 = await MixedRouteQuoterV1Factory.deploy(
+      V3_CORE_FACTORY_ADDRESSES[ChainId.MAINNET],
+      V2_FACTORY,
+      WRAPPED_NATIVE_CURRENCY[ChainId.MAINNET].address // TODO: change to be chain dependent
+    );
+
+    const MixedRouteQuoterV1Address = MixedRouteQuoterV1.address;
+
```

Comment:
Why do we deploy instead of using the one you deployed?

Reply:
Ah yes we don't need to anymore, removing in #158